crowbar_framework/config/repos-cloud.yml

```diff
@@ -73,25 +73,25 @@ suse-12.1:
       url:
       smt_path: "SUSE/Updates/SLE-HA/12-SP1/x86_64/update"
       ask_on_error: false
-    suse-enterprise-storage-2_1-pool:
-      name: "SUSE-Enterprise-Storage-2.1-Pool"
+    suse-enterprise-storage-3-pool:
+      name: "SUSE-Enterprise-Storage-3.0-Pool"
```

Comment:
But the paths and labels are inconsistent, you have both 3 and 3.0. (No idea what is correct, though)